### ISSUE TITLE
Is reverse transliteration possible?

### ISSUE THREAD

**Issue by slavaGanzin** [yf-hk/transliteration#236]
Hi. @barwin 
Awesome project, use it a lot, and it works flawlessly!

But now I had a reverse problem: I need to transliterate something from English.
My use case is translation: for example, company names or people surnames shouldn't be translated at all. 

Of course, to get flawless results graphemes need to be transcribed phonetically to IPA and then reconstructed to graphemes in target language. But it sounds like a whole a lot a work. But if you know something already working, I'll appreciate some links.

I'm not searching for flawless, I'd be happy with "it works somehow".
So, how do you think, can this be done with this library? Will it work at least in 90% of a time? 

I'm asking you, as an expert in this field, that knows obvious (not for me) caveats.

**Comment by barwin** [comment]
@slavaGanzin I agree it's a great project, but it's not mine :)  I think you mean to ping @dzcpy
